pertpy/tools/_augur.py

```diff
@@ -1034,16 +1039,24 @@ def plot_dp_scatter(
         legend1 = ax.legend(*scatter.legend_elements(), loc="center left", title="z-scores", bbox_to_anchor=(1, 0.5))
         ax.add_artist(legend1)
 
-        return fig if return_figure else ax
+        if show:
```

Comment:
These few lines are repeated everywhere and I wonder whether we can extract them to somewhere. Generally not a fan of `_util.py` of any form, but maybe you have a good idea?

Reply:
Hm, I don't have any better ideas than introducting _utils. Maybe we just can keep it as it is for now until we come up with something else. But I agree, it is kinda annoying to have copy pasta at so many places